cg/services/run_devices/pacbio/run_file_manager/run_file_manager.py

```diff
@@ -29,10 +30,22 @@ def get_files_to_store(self, run_data: PacBioRunData) -> list[Path]:
         run_path: Path = run_data.full_path
         return self.get_files_to_parse(run_data) + self._get_hifi_read_files(run_path)
 
-    @staticmethod
-    def _get_ccs_report_file(run_path: Path) -> Path:
+    @handle_post_processing_errors(
+        to_except=(FileNotFoundError,), to_raise=PostProcessingRunFileManagerError
+    )
+    def get_run_validation_files(self, run_data: PacBioRunData) -> PacBioRunValidatorFiles:
+        manifest_file: Path = self._get_manifest_file(run_data.full_path)
+        decompression_target: Path = self._get_zipped_reports_file(run_data.full_path)
```

Comment:
For me, source is more clear than target as target can be taken for destination. Feel free to disregard
```suggestion
        decompression_source: Path = self._get_zipped_reports_file(run_data.full_path)
```